mmdet/core/bbox/assigners/grid_assigner.py

```diff
@@ -26,19 +27,126 @@ class GridAssigner(BaseAssigner):
             pos_iou_thr due to the 4th step (assign max IoU sample to each gt).
         gt_max_assign_all (bool): Whether to assign all bboxes with the same
             highest overlap with some gt to that gt.
+        static (bool): Whether to statically assign a corresponding gt bbox
```

Comment:
It would be great to add unit tests to test this option. We only need to make sure that the static mode has similar output as the non-static mode in unit tests for now.